exercise14_beyond3.py

```diff
@@ -0,0 +1,9 @@
+import datetime as dt
+
+dates_of_births = {'Sergey': dt.date(1971, 5, 13), 'Tatyana': dt.date(1972, 10, 13), 'Artyom': dt.date(2005, 10, 20)}
```

Comment:
Try to develop a habit of not using real data for tests. Sharing your personal information can bring you pain in the neck one day.